router/local_peer.go

```diff
@@ -252,7 +254,7 @@ func (peer *LocalPeer) handleConnectionEstablished(conn *LocalConnection) {
 	peer.Lock()
 	peer.version += 1
 	peer.Unlock()
-	log.Println("Peer", peer.Name, "established active connection to remote peer", conn.Remote().Name, "at", conn.RemoteTCPAddr())
+	conn.log("connection fully established")
```

Comment:
you may want to take a look at the logs :)

"Attempting connection to ..."

Also, both ends log the IP/port at the end of handshake.